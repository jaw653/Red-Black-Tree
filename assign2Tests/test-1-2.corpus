j
w
q
r
v
i
r
s
w
n
b
x
f
m
r
e
h
