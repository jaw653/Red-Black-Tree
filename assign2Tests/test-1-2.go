i j
i w
i q
i r
i v
i i
i r
i s
i w
i n
i b
i x
i f
i m
i r
i e
i h
s
r
