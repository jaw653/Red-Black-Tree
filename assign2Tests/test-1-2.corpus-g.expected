0: j-2(j-2)-
1: i-2(j-2)-l w-4(j-2)-r
2: b-2(i-2)-l q-2(w-4)-l =x-2(w-4)-r
3: f-2(b-2)-r n-2(q-2)-l r-6(q-2)-r
4: =e-2(f-2)-l =h-2(f-2)-r =m-2(n-2)-l v-2(r-6)-r
5: =s-2(v-2)-l
Words/Phrases: 34
Nodes: 14
Minimum depth: 2
Maximum depth: 6
